src/platform_impl/macos/window.rs

```diff
@@ -605,6 +605,28 @@ impl UnownedWindow {
         }
     }
 
+    pub fn resize_increments(&self) -> Option<PhysicalSize<u32>> {
+        let increments = unsafe { self.ns_window.contentResizeIncrements() };
+        let (x, y) = (increments.width, increments.height);
+        if x > 1.0 || y > 1.0 {
+            Some(LogicalSize::new(x, y).to_physical(self.scale_factor()))
+        } else {
+            None
+        }
+    }
+
+    pub fn set_resize_increments(&self, increments: Option<Size>) {
+        let size = increments
+            .map(|increments| {
+                let logical = increments.to_logical::<f64>(self.scale_factor());
+                NSSize::new(logical.width.min(1.0), logical.height.min(1.0))
```

Comment:
You probably want:
```suggestion
                NSSize::new(logical.width.max(1.0), logical.height.max(1.0))
```

Since right now, clearing the resize increments works, but setting it again doesn't.